posthog/test/test_organization_model.py

```diff
@@ -61,46 +55,3 @@ def test_plugins_are_not_preinstalled_on_cloud(self, mock_get):
 
         self.assertEqual(Plugin.objects.filter(organization=new_org, is_preinstalled=True).count(), 0)
         self.assertEqual(mock_get.call_count, 0)
-
-    @pytest.mark.ee
-    @patch("posthog.models.organization.License.PLANS", {"enterprise": ["whatever"]})
-    @patch("ee.models.license.requests.post")
-    def test_feature_available_self_hosted_has_license(self, patch_post):
```

Comment:
Feedback: would've been helpful if you added a review comment here specifying you copied this verbatim to the test class in the `ee` folder